server/createExpressHandler.ts

```diff
@@ -3,10 +3,17 @@ import { Request, Response } from 'express';
 import { ServerlessContext, ServerlessFunction } from './types';
 import Twilio from 'twilio';
 
-const { TWILIO_ACCOUNT_SID, TWILIO_API_KEY_SID, TWILIO_API_KEY_SECRET, TWILIO_CONVERSATIONS_SERVICE_SID } = process.env;
+const {
+  TWILIO_ACCOUNT_SID,
+  TWILIO_API_KEY_SID,
+  TWILIO_API_KEY_SECRET,
+  TWILIO_CONVERSATIONS_SERVICE_SID,
+  REACT_APP_TWILIO_ENVIRONMENT,
+} = process.env;
 
 const twilioClient = Twilio(TWILIO_API_KEY_SID, TWILIO_API_KEY_SECRET, {
   accountSid: TWILIO_ACCOUNT_SID,
+  region: REACT_APP_TWILIO_ENVIRONMENT === 'prod' ? undefined : REACT_APP_TWILIO_ENVIRONMENT,
```

Comment:
is `undefined` one of the allowed values?

Reply:
Yeah. Here `region: 'prod'` breaks everything, but `undefined` is fine when `REACT_APP_TWILIO_ENVIRONMENT === 'prod'`